superset-frontend/src/components/Select/OnPasteSelect.jsx

```diff
@@ -75,8 +75,8 @@ export default class OnPasteSelect extends React.Component {
   }
 
   render() {
-    const { selectWrap: SelectComponent, ...restProps } = this.props;
-    return <SelectComponent {...restProps} onPaste={this.onPaste} />;
+    const { selectWrap: SelectComponent, onPaste, ...restProps } = this.props;
+    return <SelectComponent {...restProps} onPaste={onPaste || this.onPaste} />;
```

Comment:
why are we keeping around the old `this.onPaste` code? Is that actually used anywhere? Since we're adding an empty function in `AdhocFilterControl`, that will actually override the original behavior, right?

Reply:
`OnPasteSelect` is also used in `visualizations/FilterBox` and in `explore/components/SelectControl`. I did not analyse these use cases and assumed that `this.onPaste` works as expected there.